core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4212,6 +4212,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_CLIENT_SPECIAL_COMMAND_ENABLED =
```

Comment:
Change to FUSE SPECIAL COMMAND ENABLED since it's not generalized.
HDFS Client side should have their own special command